jenkins/spark-premerge-build.sh

```diff
@@ -160,6 +163,36 @@ rapids_shuffle_smoke_test() {
     $SPARK_HOME/sbin/stop-master.sh
 }
 
+# parameters for Hybrid featrue
+spark_prefix="${SPARK_VER:0:3}" # get prefix from SPARK_VER, e.g.: 3.2, 3.3 ... 3.5
+GLUTEN_BUNDLE_JAR="gluten-velox-bundle-spark${spark_prefix}_2.12-ubuntu_20.04_x86_64-1.2.0.jar"
```

Comment:
Can we def a global Gluten version `1.2.0` some where, e.g. in pom.xml?

Then we can share this version among docs and CI scripts, and easy to update it when Gluten version get updated

Reply:
Let me add a config in `version-def.sh`

GLUTEN_VERSION=1.2.0